luigi_pipeline/pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.isort]
+force_sort_within_sections = "True"
+include_trailing_comma = "True"
+known_external_package = ['hail_scripts']
```

Comment:
This is a mini-hack to lift `hail_scripts` into its own section of imports.  By default, `isort` is treating it as a `third party`.  Setting `hail_scripts` as a `first party` module seemed a bit off since those lines were getting interspersed with `lib` and `seqr_loading`.